packages/mui-material/src/Modal/ModalManager.ts

```diff
@@ -174,18 +208,32 @@ function handleContainer(containerInfo: Container, props: ManagedModalProps) {
   return restore;
 }
 
-function getHiddenSiblings(container: Element) {
+function getHiddenElements(container: Element) {
   const hiddenSiblings: Element[] = [];
-  [].forEach.call(container.children, (element: Element) => {
-    if (element.getAttribute('aria-hidden') === 'true') {
-      hiddenSiblings.push(element);
-    }
-  });
+  const html = ownerDocument(container).body.parentElement;
+  let current: Element | null = container;
```

Comment:
It would be better to change the type of `container` in these methods instead of creating a new variable called `current`.

Reply:
I think this is fine. `parentElement` can be null.